package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.15.5",
-    "tiny-emitter": "^2.1.0"
+    "tiny-emitter": "^2.1.0",
+    "unleash-proxy-client": "^1.5.0"
```

Comment:
The proxy client should not depend on itself.

Reply:
I think it was added when i run `npm install`